Wangscape/tilegen/partition/TilePartitionerNoise.cpp

```diff
@@ -45,6 +46,7 @@ noise::module::ModulePtr TilePartitionerNoise::makeCornerModule(const Corners& c
     combiner.setInputModuleSource(2, central.getOutputModule());
     if (mDebugOutput)
     {
+        setLoggingLevel(logging::Level::Debug);
```

Comment:
Yes, that's the reason and this change of a logging level is triggered by configuration option `"DebugOutput": true`, which is connected with debug tilesets. I'm not sure, what would be proper solution - we could treat that option as one that enables "developer's logging level" or change these lines below level to `Info` and treat it as user's debug info.